File: TaccomStrike.Web.UI/src/models/rest/creategamelobby.ts
export class CreateGameLobby 
{
    maxRoomLimit: number;
    gameLobbyName: string;
    gameMode: number;

    constructor() 
    {
        this.gameMode = 1;
        this.maxRoomLimit = 4;
        this.gameLobbyName = "";
    }
}

File: src/CallCheatOnline.Library.Data/ApiEntities/CreateGameLobby.cs
using System;
using System.Collections.Generic;
using System.Text;
using CallCheatOnline.Library.Data.Enums;

namespace CallCheatOnline.Library.Data.ViewModel
{
	public class CreateGameLobby
	{
		public int MaxRoomLimit { get; set; }

		public GameMode GameMode { get; set; }

		public string GameLobbyName { get; set; }
	}
}
